### PR TITLE
UnitTest

### PR DESCRIPTION
**Pull request by jananithangavel**

### CODE REVIEW

File: src/test/java/org/wso2/carbon/connector/unit/test/ldap/AuthenticateTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            templateContext.getMappedValues().put(LDAPConstants.PASSWORD, "12345");
+            messageContext.setProperty(LDAPConstants.SECURE_CONNECTION, "false");
+            messageContext.setProperty(LDAPConstants.DISABLE_SSL_CERT_CHECKING, "false");
+            Stack functionStack = new Stack();
+            functionStack.push(templateContext);
+            messageContext.setProperty("_SYNAPSE_FUNCTION_STACK", functionStack);
+            authenticate.connect(messageContext);
+            Assert.assertEquals((messageContext.getEnvelope().getBody().getFirstElement()).getFirstElement().getText(),
+                    "Success");
+
```
Comment (Kanapriya) [inline src/test/java/org/wso2/carbon/connector/unit/test/ldap/AuthenticateTest.java]: Remove this empty line and do the formatting

File: src/test/java/org/wso2/carbon/connector/unit/test/ldap/DeleteEntryTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
+package org.wso2.carbon.connector.unit.test.ldap;
+
```
Comment (Kanapriya) [inline src/test/java/org/wso2/carbon/connector/unit/test/ldap/DeleteEntryTest.java]: License header is missing

File: src/test/java/org/wso2/carbon/connector/unit/test/ldap/LdapServerSetup.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
+package org.wso2.carbon.connector.unit.test.ldap;
+
```
Comment (Kanapriya) [inline src/test/java/org/wso2/carbon/connector/unit/test/ldap/LdapServerSetup.java]: License header is missing, check this throughout the PR

Comment (jananithangavel) [reply src/test/java/org/wso2/carbon/connector/unit/test/ldap/LdapServerSetup.java]: Done

File: src/test/java/org/wso2/carbon/connector/unit/test/ldap/DeleteEntryTest.java (hunk 2 above)
Comment (jananithangavel) [reply src/test/java/org/wso2/carbon/connector/unit/test/ldap/DeleteEntryTest.java]: Done

File: src/test/java/org/wso2/carbon/connector/unit/test/ldap/AuthenticateTest.java (hunk 1 above)
Comment (jananithangavel) [reply src/test/java/org/wso2/carbon/connector/unit/test/ldap/AuthenticateTest.java]: Done

**Pull request merged**